lib/src/coq.rs

```diff
@@ -407,6 +408,36 @@ impl<'a> Expression<'a> {
                     ),
                 ]),
             ),
+            Self::MonadicApplication { func, args } => paren(
+                with_paren,
+                nest([
+                    func.to_doc(false),
+                    line(),
+                    optional_insert(!args.is_empty(), text("(||)")),
+                    optional_insert(
+                        args.is_empty(),
+                        concat([
+                            text("(|"),
+                            nest([
+                                line(),
+                                intersperse(
+                                    args.iter().map(|(param, arg)| match param {
+                                        Some(param) => render::round_brackets(group([
+                                            text(param.to_owned()),
+                                            text(" := "),
+                                            arg.to_doc(false),
+                                        ])),
+                                        None => arg.to_doc(true),
```

Comment:
```suggestion
                                        None => arg.to_doc(false),
```
I think parenthesis are not necessary here (I might be wrong).

Reply:
Yea they are indeed redundant. I'll remove them